### ISSUE TITLE
Cannot activate Developer Mode - MIB2 - A4B9 - Firmware K3344

### ISSUE THREAD

**Issue by srulz75ct** [jilleb/mib2-toolbox#258]
Hello to all and many thanks for all infos.

   I have an issue with my MIB2 unit on A4 B9 2016... cannot enable developer mode.
Few months ago i installed last update K3344 (MHI2_ER_AUG22_K3344, MU vers 1438) and apply patch through GEM and also corrected SVM error.
   All was fine, except few days before when i observed accidentally that navigation unit, during voice guidance is silent... checked all settings, incl. change language and found that in more than half of database languages are working, except UK-EN, US-EN, ES... It looks like some files are corrupted and i was trying to install last firmware again and observed that i cannot access GEM.
   I used a clone VAG-COM interface (22.3) and observed that Developer Mode cannot be activated, even after i used security password 20103 (accepted). I used another interface (Launch) with licenced software - same result (parameter out of range). I also ask for help from a certified service with licenced ODIS - online (VAS6154) and even that was not able to activate developer mode...

  Now, my question is there is any other way, except direct connection and putty to install firmware ? I think to a custom AIO Firmware that can be installed through RED Menu without GEM...

**Comment by jjff2** [comment]
change to development mode and use 20103.

**Comment by srulz75ct** [comment]
The code 20103 is not accepted by controller. Adaptation to activate developer mode gives error: value out of range. There are no any faults in any modules... scaned with VCDS, Launch and Odis. I will try with OBD 11 and Delphi but i think it is a hardware issue...

![20231110_114615](https://github.com/jilleb/mib2-toolbox/assets/150363902/198c95ac-acdc-458c-b5c1-1b1061ea8445)
![20231110_114419](https://github.com/jilleb/mib2-toolbox/assets/150363902/2a28e18e-05b0-4682-9bda-f5f1bb50f95f)
![20231110_114423](https://github.com/jilleb/mib2-toolbox/assets/150363902/4ee30370-6a72-4a88-b221-b62ef618a3a6)

**Comment by MacNee** [comment]
Have you tried S12345?

**Comment by srulz75ct** [comment]
Yes, tried both 20103 and S12345 into a clone VCDS, but also on licenced ODIS and Launch. The controller 5F is not accepting any security security-key. 

My main issue is that the unit is not working on english language (Navi & Speech System), all other languages working fine... so I intend to re-install last firmware version (thinking that some files are corrupted) and need to activate user define install through GEM.

**Comment by Jaz83** [comment]
I did one firmware update and cleared the mem fault on it and still didn't work on VCDS. I went a little higher and used vag can pro and it worked.
